day_two_guide/topics/proc_etcd_backup.adoc

```diff
@@ -139,26 +152,34 @@ Or, if using etcd the v3 API:
 40bef1f6c79b3163, started, master-0.example.com, https://192.168.55.8:2380, https://192.168.55.8:2379
 95dc17ffcce8ee29, started, master-2.example.com, https://192.168.55.13:2380, https://192.168.55.13:2379
 ----
+endif::[]
 
 [discrete]
-== Procedure
+=== Procedure
 
 [NOTE]
 ====
 While the `etcdctl backup` command is used to perform the backup, etcd v3 has
-no concept of a "backup". Instead, a "snapshot" may either be taken from a live
-member with the `etcdctl snapshot save` command or by copying the
+no concept of a "backup". Instead, you either take a "snapshot" from a live
+member with the `etcdctl snapshot save` command or copy the
 `member/snap/db` file from an etcd data directory.
 
 The `etcdctl backup` command rewrites some of the metadata contained in the
-backup (specifically, the node ID and cluster ID), which means that in the
-backup, the node loses its former identity. In order to recreate a cluster from
-the backup, a new, single-node cluster is created, then the rest of the nodes
-join the cluster. The metadata is rewritten to prevent the new node from
-inadvertently being joined onto an existing cluster.
+backup, specifically, the node ID and cluster ID, which means that in the
+backup, the node loses its former identity. To recreate a cluster from
+the backup, you create a new, single-node cluster, then add the rest of the nodes
+to the cluster. The metadata is rewritten to prevent the new node from
+inadvertently joining an existing cluster.
```

Comment:
Do we need _inadvertently_ here? 
_to prevent the new node from joining an existing cluster_